src/constants/marketplace-api.js

```diff
@@ -0,0 +1,2622 @@
+export const data = [
```

Comment:
is this data only for stories? if so, make sure that it doesn't reside in `src` because that takes up the package size